# Generated by Django 5.2.1 on 2025-05-21 19:14

import django.contrib.auth.validators
from django.db import migrations

import apps.core.fields


class Migration(migrations.Migration):

    dependencies = [
        ("users", "0002_alter_user_email_user_unique_email_case_insensitive"),
    ]

    operations = [
        migrations.RemoveConstraint(
            model_name="user",
            name="unique_email_case_insensitive",
        ),
        migrations.AlterField(
            model_name="user",
            name="email",
            field=apps.core.fields.CICharField(unique=True),
        ),
        migrations.AlterField(
            model_name="user",
            name="username",
            field=apps.core.fields.CICharField(
                error_messages={"unique": "A user with that username already exists."},
                help_text="Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.",
                max_length=150,
                unique=True,
                validators=[django.contrib.auth.validators.UnicodeUsernameValidator()],
            ),
        ),
    ]
